### ISSUE TITLE
steamshovel.qrc and symlinks (Trac #489)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#488]
From Martin Wolf:

I just encountered an unusual compilation error when compiling streamshovel in offline-software trunk when using a symlink for the steamshovel directory in the meta-project's source direcory:

make[2]: *** No rule to make target `/home/mwolf/software/meta-projects/offline-software/trunk/src/steamshovel/resources/../../phys-services/resources/doms.txt', needed by `steamshovel/qrc_steamshovel.cxx'.  Stop.
make[1]: *** [steamshovel/CMakeFiles/steamshovel.dir/all] Error 2

The problem is the relative path specification in steamshovel/resources/steamshovel.qrc line 9:

<file alias="doms.txt">../../phys-services/resources/doms.txt</file>

It would be great if an absolute path (generated by cmake) could be used instead.

Using symlinks for i3projects in meta-project's source directory is quite handy, when working with different versions. 

Migrated from https://code.icecube.wisc.edu/ticket/489
```json
{
    "status": "closed", 
    "changetime": "2015-07-22T00:49:45", 
    "description": "From Martin Wolf:\n\nI just encountered an unusual compilation error when compiling streamshovel in offline-software trunk when using a symlink for the steamshovel directory in the meta-project's source direcory:\n\nmake[2]: *** No rule to make target `/home/mwolf/software/meta-projects/offline-software/trunk/src/steamshovel/resources/../../phys-services/resources/doms.txt', needed by `steamshovel/qrc_steamshovel.cxx'.  Stop.\nmake[1]: *** [steamshovel/CMakeFiles/steamshovel.dir/all] Error 2\n\nThe problem is the relative path specification in steamshovel/resources/steamshovel.qrc line 9:\n\n<file alias=\"doms.txt\">../../phys-services/resources/doms.txt</file>\n\nIt would be great if an absolute path (generated by cmake) could be used instead.\n\nUsing symlinks for i3projects in meta-project's source directory is quite handy, when working with different versions. ", 
    "reporter": "david.schultz", 
    "cc": "david.schultz@icecube.wisc.edu", 
    "resolution": "fixed", 
    "_ts": "1437526185451006", 
    "component": "combo core", 
    "summary": "steamshovel.qrc and symlinks", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2014-03-21T18:54:41", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150211T21:59:52: 
* olivas changed status from "new" to "assigned"
* olivas changed owner from "sjackso" to "david.schultz"

**Comment by icecube-trac** [comment]
Trac update at 20150323T22:52:31: 
* olivas commented:

> steamshovel is part of offline.


* olivas changed component from "steamshovel" to "icecube offline"

**Comment by icecube-trac** [comment]
Trac update at 20150721T22:44:25: 
* david.schultz changed owner from "david.schultz" to "hdembinski"
* david.schultz changed cc from "" to "david.schultz@icecube.wisc.edu"

**Comment by icecube-trac** [comment]
Trac update at 20150722T00:49:45: 
* hdembinski commented:

> In changeset 135434:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="135434"
> pulling in doms.txt from phys-services as an svn:external to avoid a fragile relative path, closes #488
> ```


* hdembinski changed status from "assigned" to "closed"
* hdembinski changed resolution from "" to "fixed"
